util/kube/kube_util.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package kube has utility functions for interacting with kubernetes cluster using client-go library
+package kube
```

Comment:
Same comment as above. Move to cluster-essentials.

This one has _some_ merit to be here. But it's a no-go without tests.